lib/pub_sub.rb

```diff
@@ -1,4 +1,5 @@
 require 'aws-sdk'
+require 'active_support/all'
```

Comment:
unless i am mistaken it is https://github.com/westfieldlabs/pub_sub/pull/20/files#diff-f250df13a42dc9226e9a109cce695513R25

I did not add this dependency but just made it explicit. This dependency was always here